client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -32,6 +32,7 @@ class ImporterMappingPane extends React.PureComponent {
 		).isRequired,
 		sourceTitle: PropTypes.string.isRequired,
 		targetTitle: PropTypes.string.isRequired,
+		sourceType: PropTypes.string.isRequierd,
```

Comment:
Typo alert! `isRequierd`

Reply:
Woah! Nice catch. I've made it optional now anyway.